From 1f98c21723e2f38f72d3d3e000fd847555c91361 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Fri, 14 Jun 2024 05:06:24 +0000
Subject: [PATCH] TARGET_USES_64_BIT_BINDER has been deprecated.

All devices use 64-bit binder by default now. Uses of TARGET_USES_64_BIT_BINDER
should be removed

Tests Done: Above warning is removed during build.

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/cpu-arch/x86_64/BoardConfig.mk | 1 -
 groups/kernel/BoardConfig-64bit.mk    | 1 -
 groups/kernel/gmin64/BoardConfig.mk.1 | 1 -
 3 files changed, 3 deletions(-)
 delete mode 100644 groups/kernel/BoardConfig-64bit.mk
 delete mode 120000 groups/kernel/gmin64/BoardConfig.mk.1

diff --git a/groups/cpu-arch/x86_64/BoardConfig.mk b/groups/cpu-arch/x86_64/BoardConfig.mk
index 93e3190..167424a 100644
--- a/groups/cpu-arch/x86_64/BoardConfig.mk
+++ b/groups/cpu-arch/x86_64/BoardConfig.mk
@@ -7,4 +7,3 @@ TARGET_2ND_CPU_ABI := x86
 TARGET_2ND_ARCH := x86
 TARGET_2ND_ARCH_VARIANT := x86
 
-TARGET_USES_64_BIT_BINDER := true
diff --git a/groups/kernel/BoardConfig-64bit.mk b/groups/kernel/BoardConfig-64bit.mk
deleted file mode 100644
index 689e9db..0000000
--- a/groups/kernel/BoardConfig-64bit.mk
+++ /dev/null
@@ -1 +0,0 @@
-TARGET_USES_64_BIT_BINDER := true
diff --git a/groups/kernel/gmin64/BoardConfig.mk.1 b/groups/kernel/gmin64/BoardConfig.mk.1
deleted file mode 120000
index 2a11740..0000000
--- a/groups/kernel/gmin64/BoardConfig.mk.1
+++ /dev/null
@@ -1 +0,0 @@
-../BoardConfig-64bit.mk
\ No newline at end of file
-- 
2.45.2

